pkg/controllers/interface_filter_test.go

```diff
@@ -0,0 +1,73 @@
+package controllers
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "kubevirt.io/api/core/v1"
+)
+
+var _ = Describe("FilterMultusNonDefaultInterfaces", func() {
+	const defaultName = "default"
+	const nonDefaultName = "non default"
+	const nonDefaultName2 = "non default2"
+
+	It("when interfaces and networks lists are nil", func() {
+		Expect(FilterMultusNonDefaultInterfaces(nil, nil)).To(BeEmpty())
+	})
+	It("when interfaces and networks lists are empty", func() {
+		Expect(FilterMultusNonDefaultInterfaces([]v1.VirtualMachineInstanceNetworkInterface{}, []v1.Network{})).To(BeEmpty())
+	})
+	It("when there is only default network", func() {
+		Expect(FilterMultusNonDefaultInterfaces([]v1.VirtualMachineInstanceNetworkInterface{createVmInterface(defaultName)}, []v1.Network{createDefaultNetwork(defaultName)})).To(BeEmpty())
+	})
+	It("when there is a non default interface", func() {
+		nonDefaultInterface := createVmInterface(nonDefaultName)
+		result := FilterMultusNonDefaultInterfaces([]v1.VirtualMachineInstanceNetworkInterface{nonDefaultInterface, createVmInterface(defaultName)}, []v1.Network{createDefaultNetwork(defaultName), createMultusNonDefaultNetwork(nonDefaultName)})
```

Comment:
`[]v1.VirtualMachineInstanceNetworkInterface{nonDefaultInterface, createVmInterface(defaultName)}`
reverse order please (all over), so default is the first (just for better readability)
and just maybe add one test where the default is not the first one

but it is also ok like this, in order to show that order doesnt matter
if we keep it this way, maybe we need at least that one of the multi ifaces test will have the default as the first one?

Reply:
I did it this way to show the order is not important.
Moved the default to be the first one in one of the tests.